guess_the_number_project/main.py

```diff
@@ -0,0 +1,46 @@
+from random import randint
+from art import logo
+
+EASY_LEVEL_TURNS = 10
+HARD_LEVEL_TURNS = 5
```

Comment:
The convention is to capitalize variables which are directly set in the code like this. However, there's nothing special about this -- and even though it looks like you might, you actually don't access them until you assign them in line 31 (the function `set_difficulty()` returns the _variable_, and then the _value_ of that variable is assigned on line 31).

Reply:
You can't access these value inside of the function -- read or ask me more about namespaces if you're interested in understanding this more completely.

Reply:
Sure, can you tell me more about "namespaces"

Reply:
Sure! So a namespace is a section of code that has it's own variables. So you can have two functions that both have the variable `test_var` and they won't affect each other because they're in separate namespaces. In general, though, it's good to use separate variable names so that it's easy for you and others to read. This is why you have to pass variables to and from functions -- you're sharing their values (not the actual variables themselves though!) from one namespace to another.